qa-tests-backend/src/main/groovy/util/Env.groovy

```diff
@@ -26,6 +26,10 @@ class Env {
     static final GATHER_QA_TEST_DEBUG_LOGS = (System.getenv("GATHER_QA_TEST_DEBUG_LOGS") == "true")
     static final QA_TEST_DEBUG_LOGS = System.getenv("QA_TEST_DEBUG_LOGS") ?: ""
 
+    // REMOTE_CLUSTER_ARCH specifies architecture of a remote cluster on which tests are to be executed
+    // the remote cluster arch can be ppc64le or s390x, default is x86_64
+    static final HW_ARCH = System.getenv("REMOTE_CLUSTER_ARCH") ?: "x86_64"
```

Comment:
Can the same name be used as the env variable? 
```suggestion
    static final REMOTE_CLUSTER_ARCH = System.getenv("REMOTE_CLUSTER_ARCH") ?: "x86_64"
```

Reply:
changes done